assets/training/model_management/src/azureml/model/mgmt/processors/factory.py

```diff
@@ -61,11 +65,16 @@ def get_mlflow_convertor(model_framework, model_dir, output_dir, temp_dir, trans
         elif task == SupportedTasks.AUTOMATIC_SPEECH_RECOGNITION.value:
             return ASRMLflowConvertorFactory.create_mlflow_convertor(model_dir, output_dir, temp_dir, translate_params)
         # Models from Hugging face framework exported in PyFunc mlflow flavor
-        elif task in \
-                [PyFuncSupportedTasks.ZERO_SHOT_IMAGE_CLASSIFICATION.value, PyFuncSupportedTasks.EMBEDDINGS.value]:
+        elif task in [
+            PyFuncSupportedTasks.ZERO_SHOT_IMAGE_CLASSIFICATION.value, PyFuncSupportedTasks.EMBEDDINGS_CLIP.value
```

Comment:
nit: if you want you can check tags present in `preprocess_args` to identify embedding model.  This info gets passed to convertor component from download component's metadata port. It generally have these tags - 
![Uploading image.png…]()

Reply:
The png didn't work, but thanks for the info, it'll simplify things a lot.